src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
+function countAvaregeAge(people) {
+  return people.reduce((acc, currentPerson) => {
+    const currentPersonAge = currentPerson.died - currentPerson.born;
+
+    return acc + currentPersonAge / people.length;
+  }, 0);
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    const diedInCentury = men.filter(person =>
+      Math.ceil(person.died / 100) === century);
```

Comment:
```suggestion
    const diedInCentury = men.filter(
      person => Math.ceil(person.died / 100) === century);
```
Не розриваємо коллбек. Якщо переносити, то все разом